src/test/resources/noclasspath/spring/PropertyComparator.java

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Could you delete as much as possible from this file, so there remains only the code needed to reproduce error?
I do not see there what code caused problem. Or is some combination of all TypeMembers??